### PR TITLE
Release 2.7

### PR DESCRIPTION
**Pull request by ioguix**
Preparation work to release check_pga 2.7.

Please, review & comment.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+2023-09-25 v2.7:
+
+* add: compatibility with PostgreSQL 15 and 16, Jehan-Guillaume de Rorthais, Thomas Reiss
```
Comment (rjuju) [inline CHANGELOG.md:6]: I think it's missing entries for cb0e5f477986bf7b4ba4eb0a741b6a03c7b23e80 and 59fd2b9084aa4ece33316c833c3aa4124ad77ec2.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+* change: simplify session accounting in `backends_status`, Thomas Reiss
+* fix: in `pg_dump_backup`, error explicitly if given `--path` is a directory, Christophe Courtois
+* fix: `temp_files` on PG10 was counting all DB files, Christophe Courtois, Benoit Lobréau
+* fix: make `check_archiver` output `oldest_ready_wal=0` when archive queue is empty, Thomas Reiss
+* fix: make `check_archiver` work properly with PostgreSQL 10 without being superuser, Thomas Reiss
+* fix: in `backends_status`, avoid "idle in transaction" false positive for PostgreSQL 9.2 and after, Thomas Reiss
+* fix: issue with check_pgactivity missing lock file, Joern Ott, Julien Rouhaud, Jehan-Guillaume de Rorthais
+* fix: in `btree_bloat`, adjust index tuple header size, Shangzi Xie, Jehan-Guillaume de Rorthais
+
+Thank you to all contributors of this release for feedbacks, bug reports,
```
Comment (rjuju) [inline CHANGELOG.md:17]: "thank you to" feels weird.  Maybe "thanks to all..."?

File: contributors [hunk 3]
```diff
@@ -18,6 +19,7 @@ The following contributors helped to bring check_pgactivity :
   * macarbiter
   * Frédéric Yhuel
   * Julian Vanden Broeck
+  * Benoit Lobréau
```
Comment (rjuju) [inline contributors:22]: I think that Shangzi Xie should also be added here.

**Review by rjuju — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+2023-09-25 v2.7:
+
+* add: compatibility with PostgreSQL 15 and 16, Jehan-Guillaume de Rorthais, Thomas Reiss
+* change: `stat_snapshot_age` is compatible from PostgreSQL v9.5 to v14, Benoit Lobréau
+* change: simplify session accounting in `backends_status`, Thomas Reiss
+* fix: in `pg_dump_backup`, error explicitly if given `--path` is a directory, Christophe Courtois
```
Comment (Krysztophe) [inline CHANGELOG.md:9]: Not really a fix, it was working as intented, that's just user-friendliness

```suggestion
* in `pg_dump_backup`, error explicitly if `--path` is a directory, Christophe Courtois
```

File: contributors (hunk 3 above)
Comment (ioguix) [reply contributors:22]: Well, he actually reported the issue on pgsql-bloat-estimation repository. I did the fix there, then in here. So, should it be considered as a contribution to check_pga?

File: CHANGELOG.md (hunk 4 above)
Comment (ioguix) [reply CHANGELOG.md:9]: Well, we need a label, either `new`, `change` or `fix`, or manque your own :)

I decided to use `fix` as the error message was (probably) wrong before...

File: CHANGELOG.md (hunk 1 above)
Comment (ioguix) [reply CHANGELOG.md:6]: indeed for cb0e5f4, but 59fd2b9 is listed as `change: simplify session accounting in `backends_status`, Thomas Reiss`

File: CHANGELOG.md (hunk 2 above)
Comment (ioguix) [reply CHANGELOG.md:17]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (ioguix) [reply CHANGELOG.md:6]: fixed in next commit

File: contributors (hunk 3 above)
Comment (ioguix) [reply contributors:22]: OK, indeed, I forgot Shangzi Xie actually studied the code to find this issue, then reviewed my brainless patch.

**Review by frost242 — CHANGES_REQUESTED** [review]
Please, change wit to with for the backup_label_age entry.

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,7 @@ Changelog
 * add: compatibility with PostgreSQL 15 and 16, Jehan-Guillaume de Rorthais, Thomas Reiss
 * change: `stat_snapshot_age` is compatible from PostgreSQL v9.5 to v14, Benoit Lobréau
 * change: simplify session accounting in `backends_status`, Thomas Reiss
+* fix: compatibility of `backup_label_age` wit PostgreSQL 15 and after, Thomas Reiss
```
[the remark this answers is not in the document]
Comment (ioguix) [reply CHANGELOG.md]: done

**Review by frost242 — APPROVED** [review]
LGTM

**Review by rjuju — APPROVED** [review]
LGTM!

**Review by blogh — APPROVED** [review]

**Pull request merged**